index.html

```diff
@@ -0,0 +1,140 @@
+                <div class="content">
+                    <img src="Images/icons8-spotify-200.png" alt="spotify-icon">
+                    <h2>Local Musicians</h2>
+                    <p>The opening act will feature talented up-and-coming musicians from the local area, kicking off the night with their lively and energetic performances.</p>
+                </div>
+            </div>
+                <div class="col-12 custom-col-12">
+                    <div class="content">
+                        <img src="Images/icons8-singer-64.png" alt="singer">
+                        <h2>Guest Artists</h2>
+                        <p>This segment of the program will showcase a diverse range of talented artists from various countries, giving the audience a chance to experience a range of different musical styles and sounds.</p>
+                    </div>
+                </div>
+                <div class="col-12 custom-col-12">
+                    <div class="content">
+                        <img src="Images/icons8-international-music-64.png" alt="collaboration icon">
+                        <h2>Collaborative Performance</h2>
+                        <p>In this unique segment, guest artists and local musicians will come together to create a collaborative performance that showcases the power of unity and collaboration. This performance is sure to be a highlight of the night.</p>
+                    </div>
+                </div>
+                <div class="col-12 custom-col-12">
+                    <div class="content">
+                        <img src="Images/icons8-dancing-64.png" alt="dancer">
+                        <h2>Dance Intermission</h2>
+                        <p>The intermission provides a welcome break for the audience, giving them a chance to get up, stretch their legs, and dance to the beat.</p>
+                    </div>
+                </div>
+                <div class="col-12 custom-col-12">
+                    <div class="content">
+                        <img src="Images/icons8-acoustic-64.png" alt="accoustic-icon">
+                        <h2>Acoustic Sets</h2>
+                        <p>The final segment of the program will feature select artists performing intimate and acoustic sets for the audience. This is a great opportunity for the audience to get up close and personal with the artists and their music.</p>
+            </div>
+        </div>
+        </div>
+        <button>
+            Attend Harmony In Motion 2023
+        </button>
+    </section>
+    <section class="musicians">
+        <div class="featured">
+        <h3>Featured Musicians</h3>
+        <hr>
+        <div>
+            <div class="musiciansblock"></div>
+        <button id="showmore">Show More 
+            <i class="fa-solid fa-angle-down"></i>
+        </button>
+        <button id="showless">Show Less 
+            <i class="fa-solid fa-angle-up"></i>
+        </button>
+    </section>
+    <footer>
```

Comment:
- [ ] Good job working on the featured section, but there are a few things that need to be taken care of. They are listed below
- [ ] The section should have size articles that are divided into a two-column section. At the moment, you have three columns instead of two.
- [ ] Please kindly ensure that the images are of same size as that of the original design and ensure that there is a grid image just behind the main image just as it is in the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/219871205-e60a8336-142f-4775-a766-2e56445f58af.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/219871240-221a064a-c542-4e7d-bb65-b5e227f5f70f.png)